pupa/importers/vote_events.py

```diff
@@ -50,6 +50,10 @@ def get_object(self, vote_event):
 
         return self.model_class.objects.prefetch_related('votes__voter').get(**spec)
 
+    def limit_spec(self, spec):
```

Comment:
Can you remind me why this limit_spec is needed? Are events guaranteed to have a legislative session set?